app/templates/events/event_list.html

```diff
@@ -69,22 +70,62 @@ <h1 class="text-center">Events List for {{selectedTerm.description}}</h1>
     </thead>
     <tbody>
         {% for event in events %}
+        {% set Fade_event= "" %}
+        {% if event.isPast %}
+          {% if user.isCeltsStudentStaff or user.isAdmin %}
+            <tr class="table-secondary">
+              {% if type != "bonner" %}
+                {% if event.singleProgram%}
+                  <td  class=text-secondary>{{event.singleProgram.programName}}</td>
+                {% else %}
+                  <td  class=text-secondary>This event does not belong to any program.</td>
+                {% endif %}
+              {% endif %}
+              {% if user.isCeltsAdmin or user.isProgramManagerFor(event.singleProgram)%}
+                <td> <a href="/eventsList/{{event.id}}/edit" class=text-secondary>{{event.name}}</a></td>
+              {% else %}
+                <td> <a href="/eventsList/{{event.id}}/view" class=text-secondary>{{event.name}}</a></td>
+              {% endif %}
+              <td nowrap  class=text-secondary >{{event.startDate.strftime('%m/%d/%Y')}}</td>
```

Comment:
Class values should be wrapped in quotations so it would be class="text-secondary".

Reply:
This still needs resolving @qasema